package-lock.json

```diff
@@ -4658,7 +4658,8 @@
     "asap": {
       "version": "2.0.6",
       "resolved": "https://registry.npmjs.org/asap/-/asap-2.0.6.tgz",
-      "integrity": "sha1-5QNHYR1+aQlDIIu9r+vLwvuGbUY="
+      "integrity": "sha1-5QNHYR1+aQlDIIu9r+vLwvuGbUY=",
+      "dev": true
```

Comment:
All of these dependencies now have `dev: true`, are they only used for local development?

Reply:
Interesting, the `package-lock.json` file changes were auto-generated when running `npm install` so I'm not quite sure where the dev dependency markings are coming in from. Not sure if @adamstankiewicz or @abutterworth might have more insight into why this is happening.

Reply:
This is fine. My guess is that some dependencies of `prop-types` (and its own set of dependencies) were moved to `devDependencies` or `peerDependencies`, when they were previously in `dependencies` within package.json.

I wouldn't worry about these.

Reply:
Is it just production builds or does it also not install them in stage (so we can preview production behavior in a staging environment, such as catching missing dependency issues before production)?

Reply:
What you see on stage is a copy of what will be released to production; stage/prod builds are done the same way with `NODE_ENV=production`